texts/formatted_enceladus.txt

```diff
@@ -0,0 +1,1610 @@
+;	"water" vblex imp
+"<vapour>"
+	"vapour" n sg
+"<,>"
+	"," cm
+"<as well as>"
+	"as well as" cnjcoo
+"<other>"
+	"other" det ind sp
+"<substances>"
+	"substance" n pl
+"<,>"
+	"," cm
+"<erupt>"
+;	"erupt" vblex inf
+	"erupt" vblex pres
+;	"erupt" vblex imp
+"<from>"
+	"from" pr
+"<cracks>"
+	"crack" n pl
+;	"crack" vblex pres p3 sg
+"<in>"
+;	"in" adv
+	"in" pr
+"<the>"
+	"the" det def sp
+"<icy>"
+	"icy" adj sint
+"<surface>"
+	"surface" n sg
+;	"surface" vblex inf
+;	"surface" vblex pres
+;	"surface" vblex imp
+"<of>"
+	"of" pr
+"<this>"
+	"this" det dem sg
+;	"this" prn dem mf sg
+;	"this" prn rel an mf sp
+"<moon>"
+	"moon" n sg
+"<.>"
+	"." sent
+"<These>"
+	"this" det dem pl
+;	"this" prn dem mf pl
+"<plumes>"
+	"plume" n pl
+"<were>"
+	"be" vbser past
+"<first>"
+	"first" adv
+;	"first" adj ord
+"<observed>"
+;	"observe" vblex past
+	"observe" vblex pp
+"<in>"
+;	"in" adv
+	"in" pr
+"<2005>"
+	"2005" num
+"<,>"
+	"," cm
+"<when>"
+;	"when" adv
+	"when" cnjsub
+;	"when" cnjadv
+;	"when" adv itg
+"<the>"
+	"the" det def sp
+"<Cassini>"
+	"Cassini" np cog sg
+"<mission>"
+	"mission" n sg
+"<passed>"
+;	"pass" vblex pp
+	"pass" vblex past
+"<over>"
+	"over" adv
+	"over" pr
+"<Enceladus>"
+	"Enceladus" np top sg
+"<.>"
+	"." sent
+"<The>"
+	"the" det def sp
+"<passage>"
+	"passage" n sg
+"<suggested>"
+;	"suggest" vblex pp
+	"suggest" vblex past
+;	"suggested" adj
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<cryovolcanism>"
+	"cryovolcanism" n sg
+"<(>"
+	"(" lpar
+"<type>"
+	"type" n sg
+;	"type" vblex inf
+;	"type" vblex pres
+;	"type" vblex imp
+"<of>"
+	"of" pr
+"<volcanism>"
+	"volcanism" n sg
+"<where>"
+	"where" adv
+;	"where" adv itg
+"<volcanoes>"
+	"volcano" n pl
+"<erupt>"
+;	"erupt" vblex inf
+	"erupt" vblex pres
+;	"erupt" vblex imp
+"<,>"
+	"," cm
+"<instead of>"
+	"instead of" pr
+"<molten>"
+	"molten" adj
+"<rock>"
+	"rock" n sg
+;	"rock" vblex inf
+;	"rock" vblex pres
+;	"rock" vblex imp
+"<,>"
+	"," cm
+"<volatile>"
+	"volatile" adj
+"<gases>"
+	"gas" n pl
+"<such as>"
+;	"such as" adv
+	"such as" pr
+"<water>"
+	"water" n sg
+;	"water" vblex inf
+;	"water" vblex pres
+;	"water" vblex imp
+"<or>"
+	"or" cnjcoo
+"<methane>"
+	"methane" n sg
+"<)>"
+	")" rpar
+"<might>"
+;	"might" vbmod inf
+	"might" vbmod pres
+"<be>"
+	"be" vbser inf
+"<present>"
+	"present" adj
+;	"present" n sg
+;	"present" vblex inf
+;	"present" vblex pres
+;	"present" vblex imp
+"<on>"
+;	"on" adv
+	"on" pr
+"<this>"
+	"this" det dem sg
+;	"this" prn dem mf sg
+;	"this" prn rel an mf sp
+"<planet>"
+	"planet" n sg
+"<,>"
+	"," cm
+"<even though>"
+	"even though" cnjadv
+"<scientists>"
+	"scientist" n pl
+"<are>"
+	"be" vbser pres
+"<unsure>"
+	"unsure" adj
+"<,>"
+	"," cm
+"<with>"
+	"with" pr
+"<some>"
+;	"some" adj
+;	"some" adv
+;	"some" det qnt sp
+	"some" prn ind mf pl
+"<believing>"
+	"believing" adj
+	"believe" vblex pprs
+	"believe" vblex subs
+	"believe" vblex ger
+"<they’re>"
```

Comment:
These 2 words are not disambiguated

Reply:
Two words? I'm guessing you mean `they're` as well, but it has two entries because it's made up of two words. That's why the second one is further inline